index.html

```diff
@@ -0,0 +1,123 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <title>review-own-code</title>
+</head>
+
+<body>
+    <div id="main">
+        <h1 id="title">Dr.Norman Borlaug</h1>
```

Comment:
the appropriate semantic tag for this line would be "title" tag.

Reply:
- [ ] As I commented above your comment should be phrased politely, as suggestions instead of orders, and always tell the reason WHY. :+1:  Please fix it :+1:

Reply:
@codecaiine  okay bro. i will comment politely as suggestion.

Reply:
I would suggest the usage of semantic tag (title) for this line instead of h1 because using of semantic tags helps in SEO and also it would be better for clean code.